src/value.rs

```diff
@@ -569,32 +652,60 @@ pub fn sea_value_to_json_value(value: &Value) -> Json {
     use crate::{PostgresQueryBuilder, QueryBuilder};
 
     match value {
-        Value::Null => Json::Null,
-        Value::Bool(b) => Json::Bool(*b),
-        Value::TinyInt(v) => (*v).into(),
-        Value::SmallInt(v) => (*v).into(),
-        Value::Int(v) => (*v).into(),
-        Value::BigInt(v) => (*v).into(),
-        Value::TinyUnsigned(v) => (*v).into(),
-        Value::SmallUnsigned(v) => (*v).into(),
-        Value::Unsigned(v) => (*v).into(),
-        Value::BigUnsigned(v) => (*v).into(),
-        Value::Float(v) => (*v).into(),
-        Value::Double(v) => (*v).into(),
-        Value::String(s) => Json::String(s.as_ref().clone()),
-        Value::Bytes(s) => Json::String(from_utf8(s).unwrap().to_string()),
-        Value::Json(v) => v.as_ref().clone(),
+        Value::Bool(None)
+        | Value::TinyInt(None)
+        | Value::SmallInt(None)
+        | Value::Int(None)
+        | Value::BigInt(None)
+        | Value::TinyUnsigned(None)
+        | Value::SmallUnsigned(None)
+        | Value::Unsigned(None)
+        | Value::BigUnsigned(None)
+        | Value::Float(None)
+        | Value::Double(None)
+        | Value::String(None)
+        | Value::Bytes(None)
+        | Value::Json(None) => Json::Null,
+        #[cfg(feature = "with-rust_decimal")]
+        Value::Decimal(None) => Json::Null,
+        #[cfg(feature = "with-bigdecimal")]
+        Value::BigDecimal(None) => Json::Null,
+        #[cfg(feature = "with-uuid")]
+        Value::Uuid(None) => Json::Null,
+        Value::Bool(Some(b)) => Json::Bool(*b),
+        Value::TinyInt(Some(v)) => (*v).into(),
+        Value::SmallInt(Some(v)) => (*v).into(),
+        Value::Int(Some(v)) => (*v).into(),
+        Value::BigInt(Some(v)) => (*v).into(),
+        Value::TinyUnsigned(Some(v)) => (*v).into(),
+        Value::SmallUnsigned(Some(v)) => (*v).into(),
+        Value::Unsigned(Some(v)) => (*v).into(),
+        Value::BigUnsigned(Some(v)) => (*v).into(),
+        Value::Float(Some(v)) => (*v).into(),
+        Value::Double(Some(v)) => (*v).into(),
+        Value::String(Some(s)) => Json::String(s.as_ref().clone()),
+        Value::Bytes(Some(s)) => Json::String(from_utf8(s).unwrap().to_string()),
+        Value::Json(Some(v)) => v.as_ref().clone(),
+        #[cfg(feature = "with-chrono")]
+        Value::Date(_) => PostgresQueryBuilder.value_to_string(value).into(),
+        #[cfg(feature = "with-chrono")]
+        Value::Time(_) => PostgresQueryBuilder.value_to_string(value).into(),
         #[cfg(feature = "with-chrono")]
         Value::DateTime(_) => PostgresQueryBuilder.value_to_string(value).into(),
         #[cfg(feature = "with-chrono")]
         Value::DateTimeWithTimeZone(_) => PostgresQueryBuilder.value_to_string(value).into(),
```

Comment:
I realized it cannot compile if the Postgres backend is not enabled. I will refactor it later.

Reply:
Thanks!!!